x-pack/plugins/security_solution/public/onboarding/constants.ts

```diff
@@ -6,11 +6,30 @@
  */
 export const PAGE_CONTENT_WIDTH = '1150px';
 
+export enum OnboardingTopicId {
+  default = 'default',
+  siemMigrations = 'siem_migrations',
+}
+
 export enum OnboardingCardId {
   integrations = 'integrations',
   dashboards = 'dashboards',
   rules = 'rules',
   alerts = 'alerts',
   assistant = 'assistant',
   attackDiscovery = 'attack_discovery',
+
+  // siem_migrations topic cards
+  siemMigrationsAiConnectors = 'ai_connectors',
 }
+
+export const LocalStorageKey = {
+  avcBannerDismissed: 'ONBOARDING_HUB.AVC_BANNER_DISMISSED',
+  videoVisited: 'ONBOARDING_HUB.VIDEO_VISITED',
+  completeCards: 'ONBOARDING_HUB.COMPLETE_CARDS',
+  expandedCard: 'ONBOARDING_HUB.EXPANDED_CARD',
+  urlDetails: 'ONBOARDING_HUB.URL_DETAILS',
+  selectedIntegrationTabId: 'ONBOARDING_HUB.SELECTED_INTEGRATION_TAB_ID',
+  integrationSearchTerm: 'ONBOARDING_HUB.INTEGRATION_SEARCH_TERM',
+  siemMigrationsConnectorId: 'ONBOARDING_HUB.SIEM_MIGRATIONS_CONNECTOR_ID',
+} as const;
```

Comment:
Could we add the "topic" to the StorageKeys, and use the convention in the new introduced Key?

Like `siemMigrationsConnectorId: 'securitySolution.onboarding.siemMigrations.connectorId',`

Reply:
Yes, that makes sense. I changed all the keys to lowercase, which is the convention across Security.
I also realized the `siemMigrationsConnectorId` key is not necessary anymore, the storage is managed by the `siemMigration` service now. And all the keys are only used inside the _use_stored_state.ts_ file again, so I moved the keys back to this file, without exporting them.